torch/_prims/__init__.py

```diff
@@ -1912,3 +1838,143 @@ def _make_bool_reduction_prim(name: str, impl_aten, doc):
     impl_aten=torch.any,
     doc="",
 )
+
+# TODO: layout, pin_memory, memory_format
```

Comment:
Somewhat surprised the meta tests aren't complaining loudly at you on this ;)

Reply:
The samples don't set these options